lib/lobby.ts

```diff
@@ -0,0 +1,176 @@
+import { getClient } from '.';
+import { createSpotifyPlaylist } from '../utils/createSpotifyPlaylist';
+import { DbItem, IDbItem } from './db-item';
+import { COLLECTION } from './private/enums';
+
+type DatabaseEntry = Omit<ILobby, 'collectionName'>;
+type ClientResponse = Omit<DatabaseEntry, 'users' | 'uri'>;
+
+export interface LobbyCreateProps {
+  /**
+    * The manager of a lobby
+    */
+  readonly managerId: string;
+
+  /**
+   * The theme of a lobby
+   */
+  readonly theme: string;
+
+  /**
+    * The playlist name
+    */
+  readonly name: string;
+}
+
+export interface LobbyProps extends LobbyCreateProps{
+  /**
+  * The spotify id of playlist
+  */
+  readonly spotifyPlaylistId: string;
```

Comment:
me thinks this doesnt need to be in lobby props since its just the id

Reply:
instead we should then put it in ILobby

Reply:
Oh I just tried and the id is needed in lobby props for the constructor

Reply:
Thats Bc ur using the prop.. just use the id instead?